src/ui/server/helpers/sanitise-data/index.ts

```diff
@@ -48,6 +49,20 @@ const sanitiseValue = (value: string | number | boolean) => {
   return value;
 };
 
+/**
+ * isDayMonthYearField
+ * Checks if a field is a day, month or year field
+ * @param {String} Form field name
+ * @returns {Boolean}
+ */
+const isDayMonthYearField = (fieldName: string): boolean => {
+  if (fieldName.includes('-day') || fieldName.includes('-month') || fieldName.includes('-year')) {
```

Comment:
Do we need to add an empty `fieldName` check or is this never be the case?

Reply:
Same comment as above